backend/code_review_backend/version.json

```diff
@@ -0,0 +1,6 @@
+{
+  "commit": "stub",
```

Comment:
You should be able to add a simple unit test checking that endpoint (checking the heartbeat should be possible too).